token-lending/cli/src/main.rs

```diff
@@ -0,0 +1,700 @@
+                )
+                .arg(
+                    Arg::with_name("oracle_program_id")
+                        .long("oracle")
+                        .validator(is_pubkey)
+                        .value_name("PUBKEY")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Oracle (Pyth) program ID for quoting market prices"),
+                )
+                .arg(
+                    Arg::with_name("quote_currency")
+                        .long("quote")
+                        .value_name("CURRENCY")
+                        .takes_value(true)
+                        .required(true)
+                        .default_value("USD")
+                        .help("Currency market prices are quoted in"),
+                ),
+        )
+        .subcommand(
+            SubCommand::with_name("add-reserve")
+                .about("Add a reserve to a lending market")
+                .arg(
+                    Arg::with_name("lending_market")
+                        .long("market")
+                        .validator(is_pubkey)
+                        .value_name("PUBKEY")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Lending market address"),
+                )
+                .arg(
+                    Arg::with_name("lending_market_owner")
+                        .long("owner")
+                        .validator(is_keypair)
+                        .value_name("KEYPAIR")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Owner required to sign when adding reserves to the lending market"),
+                )
+                .arg(
+                    Arg::with_name("source_liquidity")
+                        .long("source")
+                        .validator(is_pubkey)
+                        .value_name("PUBKEY")
+                        .takes_value(true)
+                        .required(true)
+                        .help("SPL Token account to deposit initial liquidity from"),
+                )
+                .arg(
+                    Arg::with_name("liquidity_amount")
+                        .long("amount")
+                        .validator(is_amount)
+                        .value_name("AMOUNT")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Initial amount of liquidity to deposit into the new reserve"),
+                )
+                .arg(
+                    Arg::with_name("pyth_product")
+                        .long("pyth-product")
+                        .validator(is_pubkey)
+                        .value_name("PUBKEY")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Pyth product account"),
```

Comment:
Can you add instructions on where a user would be look up which account address to use?

Reply:
Done.